src/shell/components/shell-view.scss

```diff
@@ -97,12 +102,7 @@ $table-background-alt-color: #fafafa;
     thead {
         .table-col-title-group th {
             border-bottom: $table-border-width solid $table-border-color;
-        }
-        
-        th:first-child .table-col-title,
-        .table-col-featured .table-col-title {
-            border-left: 0px;
-        }
+        }        
```

Comment:
Why was this removed? It prevents the left side boarder from showing in the first column.